src/main/scala/viper/gobra/Gobra.scala

```diff
@@ -266,7 +266,10 @@ class Gobra extends GoVerifier with GoIdeVerifier {
     * be easily extended to perform more transformations
     */
   private def performInternalTransformations(program: Program, config: Config, pkgInfo: PackageInfo): Either[Vector[VerifierError], Program] = {
-    var transformations: Vector[InternalTransform] = Vector(CGEdgesTerminationTransform)
+    // having constant propagation before overflow checking will not lead to duplication of errors if
+    // overflow checks are enabled, given that all overflows in constant declarations can be found by the
+    // well-formedness checks
```

Comment:
```suggestion
    // constant propagation does not cause duplication of verification errors caused
    // by overflow checks (if enabled) because all overflows in constant declarations 
    // can be found by the well-formedness checks.
```